resources/ApprovePage.js

```diff
@@ -22,47 +21,48 @@ const handleApprovalResponse = ( approve, data ) => {
 		'.approver-details',
 		'approval-dropdown'
 	];
-	const badgeElement = $( '.approval-status-badge' );
+	const $badgeElement = $( '.approval-status-badge' );
 
-	badgeElement.toggleClass( 'approved-badge' );
-	badgeElement.toggleClass( 'unapproved-badge' );
+	$badgeElement.toggleClass( 'approved-badge' );
```

Comment:
Does eslint enforce us to define elemental variables with $?

Reply:
Yes. There was already another element variable where we were using it, so I figured it's fine. We can of course disable it, but I don't think it's a big deal and it's been a JQuery convention for long.